tests/space-time-stack/test_demangling.cpp

```diff
@@ -0,0 +1,90 @@
+#include <iostream>
+#include <regex>
+#include <sstream>
+
+#include "Kokkos_Core.hpp"
+#include "kp_all.hpp"
+
+#include "utils/demangle.hpp"
+
+template <typename execution_space, size_t size = 5>
+struct Tester {
+  struct TagNamed {};
+  struct TagUnnamed {};
+
+  void apply(const execution_space& space) const {
+    //! Explicitly launch a kernel with a name and no tag.
+    Kokkos::parallel_for("named kernel",
+                         Kokkos::RangePolicy<execution_space>(space, 0, size),
+                         *this);
+
+    //! Explicitly launch a kernel with a name and a tag.
+    Kokkos::parallel_for(
+        "named kernel",
+        Kokkos::RangePolicy<execution_space, TagNamed>(space, 0, size), *this);
+
+    //! Explicitly launch a kernel with no name and no tag.
+    Kokkos::parallel_for(Kokkos::RangePolicy<execution_space>(space, 0, size),
+                         *this);
+
+    //! Explicitly launch a kernel with no name and a tag.
+    Kokkos::parallel_for(
+        Kokkos::RangePolicy<execution_space, TagUnnamed>(space, 0, size),
+        *this);
+  }
+
+  KOKKOS_INLINE_FUNCTION
+  void operator()(const int index) const {}
```

Comment:
```suggestion
  KOKKOS_FUNCTION void operator()(int) const {}
```

Reply:
OK. It's indeed implicitly inline since it is defined in the *struct* itself. Done :heavy_check_mark:

Reply:
But why would you remove the *const* ?

Reply:
Because the const does nothing.  You don't use the argument.